src/main/java/com/google/cloud/genomics/utils/GenomicsUtils.java

```diff
@@ -0,0 +1,89 @@
+package com.google.cloud.genomics.utils;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.List;
+
+import com.google.api.services.genomics.Genomics;
+import com.google.api.services.genomics.model.CallSet;
+import com.google.api.services.genomics.model.ReadGroupSet;
+import com.google.api.services.genomics.model.ReferenceBound;
+import com.google.api.services.genomics.model.SearchCallSetsRequest;
+import com.google.api.services.genomics.model.SearchReadGroupSetsRequest;
+import com.google.api.services.genomics.model.SearchVariantSetsRequest;
+import com.google.api.services.genomics.model.VariantSet;
+import com.google.common.collect.Lists;
+
+/**
+ * Convenience routines for fetching ids in the hierarchy of data within the Genomics API and other data lookups.
+ */
+public class GenomicsUtils {
+  
+  /**
+   * Gets ReadGroupSetIds from a given datasetId using the Genomics API.
+   */
```

Comment:
perhaps add: "Throws an exception if there is none", since this behavior may be surprising to some (reading from files or console doesn't throw if the input is empty).